src/commonMain/kotlin/app/cash/turbine/ReceiveTurbine.kt

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.turbine
+
+import kotlinx.coroutines.channels.ReceiveChannel
+
+public interface ReceiveTurbine<T> {
+  /**
+   * Returns the underlying [ReceiveChannel].
+   */
+  public fun asChannel(): ReceiveChannel<T>
+
+  /**
+   * Yields true if remaining events for this test channel have been ignored.
```

Comment:
```suggestion
   * True if remaining events for this test channel have been ignored.
```

Just avoiding coroutine terminology to prevent any potential confusion